python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md

```diff
@@ -95,7 +95,7 @@ cd /PATH/TO/DOWNLOAD/MODEL/
 wget http://paddle-inference-dist.bj.bcebos.com/int8/${MODEL_FILE_NAME}
 ```
 
-Unzip the downloaded model to the folder.To verify all the 7 models, you need to set `MODEL_NAME` to one of the following values in command line:
+Unzip the downloaded model to the folder. To verify all the 7 models, you need to set `MODEL_NAME` to one of the following values in command line:
 ```text
 QAT MKL-DNN 1.0
```

Comment:
IMO, `QAT MKL-DNN 2.0` is a bit misleading, at first I thought about MKL-DNN 2.0. I would suggest using
```text
QAT1.0 MKL-DNN
...
QAT2.0 MKL-DNN
...
```
here and below.